{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "       AssertionError: Tracking Identifiier is not present for ['acute_aortic_syndrome'] observation in FHIR\n       assert ['acute_aortic_syndrome'] == True\n       Failed: Test failed since Tracking Identifier is absent for ['acute_aortic_syndrome'] observations in FHIR.json\n", "test_name": "test_fhir_tracking_id", "rerun": "0"}}, "suite_name": "tests/cxr/fhir_validation/test_fhir.py"}}}, "date": "January 30, 2024", "start_time": 1706617636.9831653, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}